platform/darwin/src/MGLNetworkConfiguration.m

```diff
@@ -48,4 +55,53 @@ - (NSURLSessionConfiguration *)defaultSessionConfiguration {
     return sessionConfiguration;
 }
 
+- (void)startDownloadEvent:(NSString *)urlString type:(NSString *)resourceType {
+    if (urlString && ![self.events objectForKey:urlString]) {
+        [self.events setObject:@{ MGLStartTime: [NSDate date], MGLResourceType: resourceType } forKey:urlString];
```

Comment:
Is it possible to have multiple events with the same `urlString`?

Reply:
At different times yes.

Reply:
But not overlapping?

Reply:
No at the same time. You may start downloading a tile and if is not cached then you will download that tile again but at a different time.